### PR TITLE
Permission changes for study coordinators

### PR DESCRIPTION
**Pull request by alxdarksage**
One of the bigger changes is that participants can get any study under their ID irrespective of their current enrollments...an edge case that will probably never come up.

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/AssessmentResourceTest.java [hunk 1]
```diff
@@ -6,13 +6,11 @@
 import static org.junit.Assert.assertNotEquals;
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/AssessmentResourceTest.java:6]: Why were these tests entirely removed, and not updated for clarity like the next test?

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/AssessmentTest.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ public void before() throws Exception {
         orgsApi.addMember(ORG_ID_2, otherDeveloper.getUserId()).execute();
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/AssessmentTest.java:92]: Org membership shouldn't matter if you're a developer, but it should if you're a study designer. So what's happening here?

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/AssessmentTest.java [hunk 3]
```diff
@@ -256,13 +238,6 @@ public void test() throws Exception {
         // deleteAssessment physical=false works
         assessmentApi.deleteAssessment(secondRevision.getGuid(), false).execute();
         
-        // deleteAssessment physical=false fails for developer who doesn't own the assessment
-        try {
-            badDevApi.deleteAssessment(secondRevision.getGuid(), false).execute();
-            fail("Should have thrown an exception");
-        } catch(EntityNotFoundException e) {
-        }
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/AssessmentTest.java:264]: So verify that the ownerId was changed to the caller's organization
